Reqnroll.VisualStudio/Editor/Services/Formatting/GherkinDocumentFormatter.cs

```diff
@@ -140,7 +155,8 @@ public void FormatTable(DocumentLinesEditBuffer lines, IHasRows hasRows, Gherkin
             foreach (var item in row.Cells.Select((c, i) => new {c, i}))
             {
                 result.Append(formatSettings.TableCellPadding);
-                result.Append(EscapeTableCellValue(item.c.Value).PadRight(widths[item.i]));
+                var paddedCell = IsTableCellContentRightAligned(item.c.Value, formatSettings) ? EscapeTableCellValue(item.c.Value).PadLeft(widths[item.i]) : EscapeTableCellValue(item.c.Value).PadRight(widths[item.i]);
```

Comment:
suggestion to make it easier to read:
```suggestion
                var escapedCellValue = EscapeTableCellValue(item.c.Value);
                var width = widths[item.i];
                var paddedCell = IsTableCellContentRightAligned(item.c.Value, formatSettings) ? escapedCellValue.PadLeft(width) : escapedCellValue.PadRight(width);
```